PublicationsRouter/v2/api/API.md

```diff
@@ -168,19 +186,89 @@ The system will not attempt to aggressively validate the request, but the reques
 
 On a successful call to this endpoint, your notification will be accepted into PubRouter where it will be queued for subsequent processing and routing to matched repositories.
 
-#### Metadata-only request
+### 1. Notification Metadata-only request
```

Comment:
Apply comments from the Validation section to the following.  (e.g. use of [] brackets and the point about position of response codes section).